widget.css

```diff
@@ -1,5 +1,5 @@
 body {
-  background: #787878;
+  background: black;
```

Comment:
Why the black for the background? It's looks not so friendly for the user.

Reply:
I think we have an issue there because the body color takes part from the wrapper class. I can tackle it in another PR : )

Reply:
I chose to make the background black because the gradient on the tile fades to black. Maybe it's not an issue anymore, but when the iframe was larger than the size of the content, it would cut from black to gray. This can happen when something is off with the iframe-resizer js and I wanted to prevent that visual glitch in that case

Reply:
I don't care much about how the index page looks like. It's only intended for testing the scripts